### DISCUSSION TITLE
Shortcut for audio output switching

### DISCUSSION THREAD

**Discussion by MBombeck** [File-New-Project/EarTrumpet#1047] [Ideas]
It would be really great if EarTrumpet could switch the audio output via shortcut. If I'm in a Fullscreen-Application right-clicking the notification area is not possible / unnecessarily slow.

**Comment by subjut** [comment]
It's currently possible to switch the default audio output with Actions in the Settings menu (dev build). Define a hotkey as trigger, set to which output you wish to switch and under advanced filters, define the currently used output. For switching back, you define another action with the outputs reversed. You can even use the same hotkey for both actions.

The only downside with this approach is, that you need to define an action for every output device, otherwise you cannot switch back. And the prerequisite is, that the applications you want to switch are set to use the default output.

**Reply by subjut** [reply]
But what I would like is an option/filter to only switch the application with the active window to another output.
